npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater.rb

```diff
@@ -147,7 +147,8 @@ def run_yarn_updater(path:, yarn_lock:, top_level_dependency_updates:)
           retry_count += 1
           raise if retry_count > 2
 
-          sleep(rand(3.0..10.0)) && retry
+          sleep(rand(3.0..10.0))
+          retry
```

Comment:
This never worked. `sleep` returns an integer, which is coerced into a truthy value, so the right hand side of the `&&` is never evaluated.

Reply:
When considering typing, I definitely think this is the right change 💯

Separately: noting that the right-hand side _will_ be evaluated in this situation:

```plaintext
irb(main):001> sleep(1) && 42
=> 42
```

Reply:
Interesting... Sorbet led me to believe that the right-hand side was dead code